app/controllers/declaration/AdditionalDeclarationTypeController.scala

```diff
@@ -66,11 +66,11 @@ class AdditionalDeclarationTypeController @Inject()(
       case StandardDec      => AdditionalDeclarationTypeStandardDec
     }
 
-  private def updateCache(sessionId: String, formData: AdditionalDeclarationType)(
+  private def updateCache(formData: AdditionalDeclarationType)(
     implicit request: JourneyRequest[_]
   ): Future[Option[ExportsDeclaration]] =
-    updateExportsDeclaration(model => {
-      exportsCacheService.update(sessionId, model.copy(additionalDeclarationType = Some(formData)))
+    updateExportsDeclarationSyncDirect(model => {
```

Comment:
For whatever reason we use this `updateExportsDeclaration` method a lot with a call to `exportsCacheService.update` inside. `updateExportsDeclaration` already has this service call inside it so we've been calling `exportsCacheService.update` twice all the time. This is fixed now.